src/foam/nanos/crunch/Capability.js

```diff
@@ -57,8 +58,12 @@ foam.CLASS({
     {
       name: 'id',
       class: 'String',
-      updateVisibility: 'RO'
+      visibility: 'HIDDEN'
```

Comment:
RO please.   This will end up in the defaultSection.

Reply:
changed to only be hidden on create